src/pages/search/articles.tsx

```diff
@@ -0,0 +1,110 @@
+import { Container, Stack, Text, createStyles } from '@mantine/core';
+import {
+  InstantSearch,
+  SearchBox,
+  useInfiniteHits,
+  useInstantSearch,
+} from 'react-instantsearch-hooks-web';
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch';
+
+import { env } from '~/env/client.mjs';
+import {
+  SearchableMultiSelectRefinementList,
+  SortBy,
+} from '~/components/Search/CustomSearchComponents';
+import { routing } from '~/components/Search/useSearchState';
+import { useInView } from 'react-intersection-observer';
+import { useEffect } from 'react';
+import { SearchHeader } from '~/components/Search/SearchHeader';
+
+const searchClient = instantMeiliSearch(
+  env.NEXT_PUBLIC_SEARCH_HOST as string,
+  env.NEXT_PUBLIC_SEARCH_CLIENT_KEY,
+  { primaryKey: 'id', keepZeroFacets: true }
+);
+
+export default function Search() {
+  return (
+    <InstantSearch searchClient={searchClient} indexName="articles" routing={routing}>
+      <Container fluid>
+        <Stack
+          sx={(theme) => ({
+            height: 'calc(100vh - 2 * var(--mantine-header-height,50px))',
+            position: 'fixed',
+            left: 0,
+            top: 'var(--mantine-header-height,50px)',
+            width: '377px',
+            overflowY: 'auto',
+            padding: theme.spacing.md,
+          })}
+        >
+          <RenderFilters />
+        </Stack>
+
+        <Stack pl={377} w="100%">
+          <SearchHeader />
+          <ArticlesHitList />
+        </Stack>
```

Comment:
Wouldn't it make sense to use `<Grid>` in here to maintain consistency between breakpoints? Having fixed width feels weird

Reply:
Yeah some improvements to be there, but I don't think Grid is the right call here. 
Technically, we do want a fixed grid for the filters. And we should move the items to the side that amount except for mobile.